iot_relay_server/app/ai_agent_prompts.py

```
@@ -1,79 +1,91 @@
-        forecast_text += (
-            f"  - {fc.get('hours_ahead', '?')}시간 후: "
-            f"기온 {fc.get('temperature', '?')}C, 습도 {fc.get('humidity', '?')}%, "
-            f"하늘 {fc.get('sky', '?')}, 강수확률 {fc.get('precipitation_prob', 0)}%\n"
-        )
 
-    return f"""## 현재 센서값
-- 온도: {sensor_data.get('temperature', 0)}C (신뢰도: {reliability.get('temperature', 'reliable')})
-- 습도: {sensor_data.get('humidity', 0)}% (신뢰도: {reliability.get('humidity', 'reliable')})
-- 조도: {sensor_data.get('light_intensity', 0)} lux (신뢰도: {reliability.get('light_intensity', 'reliable')})
-- 토양수분: {sensor_data.get('soil_moisture', 0)}%
+def build_trigger_prompt(sensor_data: dict, reliability: dict, weather: dict = None, control_st: dict = None, crop_prof: dict = None) -> str:
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)